frontend/device/kobo/device.lua

```diff
@@ -534,6 +536,83 @@ local KoboCondor = Kobo:extend{
     isSMP = yes,
 }
 
+-- Kobo Clara B/W:
+local KoboSpaBW = Kobo:extend{
+    model = "Kobo_spaBW",
+    isMk7 = yes,
```

Comment:
This needs to go (it's irrelevant for MTK).

Reply:
This seems true, there is no difference that I see on my device with or without.